### PR TITLE
issue-88: Links in 15-minute series 404

### LINKED ISSUE #88: Examples in the 15-minute series link to old KIE repo and are 404ing
When doing the 15-minute examples, several are 404-ing with the move to the Apache-incubator repository. This can be seen in https://www.ibm.com/docs/en/ibamoe/9.0.x?topic=started-orchestrating-decisions-rules-in-15-minutes which points to https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/process-business-rules-quarkus but this will return a 404 on the particular identified project. This should be at https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/process-business-rules-quarkus

The linked project should probably also be to a particular branch that uses the IBM build of Kogito for said example versus the stable branch using most current.

### PR DESCRIPTION
**Pull request by LightGuard**
- Fixes #88 
- **Fixing the github repo location**

### CODE REVIEW

File: getting-started/orchestrating.adoc [hunk 1]
```diff
@@ -160,7 +160,7 @@ image::traffic-violation-dmn-businessrule.png[Image of traffic violation task (D
 The properties that are required to set for the DMN service task include `Rule Language`, which must be set as `DMN`. Also, you must set the values for `Namespace`, `Decision Name` and DMN Model Name` properties as defined in the DMN model, such as `TrafficViolation.dmn`. For example, you can set the following values:
 
 - Rule Language: `DMN`
-- Namespace: `https://github.com/kiegroup/drools/kie-dmn/_A4BCA8B8-CF08-433F-93B2-A2598F19ECFF`
+- Namespace: `https://github.com/apache/incubator-kie-drools/kie-dmn/_A4BCA8B8-CF08-433F-93B2-A2598F19ECFF`
```
Comment (martinweiler) [inline getting-started/orchestrating.adoc:163]: I don't see the namespace being changed anywhere else in the incubator repos, so I'd leave it as-is here as well.

File: runtime-services-modeling/deploying-services.adoc [hunk 2]
```diff
@@ -547,7 +547,7 @@ Any dependant add-on can implement the https://github.com/kiegroup/kogito-runtim
 . Enter the full name of your implementation class in the file.
 . Save the file.
 
-The `MessagePayloadDecoratorProvider` loads the file upon application startup and adds the file to the decoration chain. When {PRODUCT} calls the https://github.com/kiegroup/kogito-runtimes/blob/main/addons/common/messaging/common/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecoratorProvider.java[MessagePayloadDecoratorProvider#decorate], your implementation is part of the decoration algorithm.
+The `MessagePayloadDecoratorProvider` loads the file upon application startup and adds the file to the decoration chain. When {PRODUCT} calls the https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/addons/common/messaging/common/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecoratorProvider.java[MessagePayloadDecoratorProvider#decorate], your implementation is part of the decoration algorithm.
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:550]: This class has moved, new location is:
https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/addons/common/messaging/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecorator.java

File: runtime-services-modeling/deploying-services.adoc [hunk 3]
```diff
@@ -535,7 +535,7 @@ The mail add-on is used to send emails in a process project. To use the mail add
 The events add-on provides a default implementation in supported target platforms for EventEmitter and EventReceiver interfaces. You can use EventEmitter and EventReceiver interfaces to enable messaging by process, serverless workflow events, and event decision handling. 
 
 ==== Implementing message payload decorator
-Any dependant add-on can implement the https://github.com/kiegroup/kogito-runtimes/blob/main/addons/common/messaging/common/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecorator.java[MessagePayloadDecorator].
+Any dependant add-on can implement the https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/addons/common/messaging/common/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecorator.java[MessagePayloadDecorator].
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:538]: This class has moved, new location is:
https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/addons/common/messaging/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecorator.java

File: runtime-services-modeling/decisions-with-dmn.adoc [hunk 4]
```diff
@@ -699,7 +699,7 @@ Some boxed expressions are more complex with greater detail and calculation, suc
 .Boxed function expression for passenger reassignment
 image::dmn-reassign-passenger.png[]
 
-You can find the DMN source file for this decision model on https://raw.githubusercontent.com/kiegroup/drools/8.39.0.Final/kie-dmn/kie-dmn-core/src/test/resources/org/kie/dmn/core/0019-flight-rebooking.dmn[**github**] .
+You can find the DMN source file for this decision model on https://raw.githubusercontent.com/apache/incubator-kie-drools/8.39.0.Final/kie-dmn/kie-dmn-core/src/test/resources/org/kie/dmn/core/0019-flight-rebooking.dmn[**github**] .
```
Comment (martinweiler) [inline runtime-services-modeling/decisions-with-dmn.adoc:702]: use properties here:
https://raw.githubusercontent.com/apache/incubator-kie-drools/{DROOLS_VERSION}/kie-dmn/kie-dmn-core/src/test/resources/org/kie/dmn/core/0019-flight-rebooking.dmn

File: runtime-services-modeling/deploying-services.adoc [hunk 5]
```diff
@@ -302,14 +302,14 @@ If the topic property is not found, then the topic name is considered to be the
 
 ==== Providing a key for a Kafka record
 
-To specify the key to your Kafka record, you can use a custom https://github.com/kiegroup/kogito-runtimes/blob/main/quarkus/addons/messaging/common/src/main/java/org/kie/kogito/addon/quarkus/messaging/common/message/MessageDecorator.java[MessageDecorator] implementation. 
+To specify the key to your Kafka record, you can use a custom https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/quarkus/addons/messaging/common/src/main/java/org/kie/kogito/addon/quarkus/messaging/common/message/MessageDecorator.java[MessageDecorator] implementation.
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:305]: new location:
https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/quarkus/addons/common/reactive-messaging/src/main/java/org/kie/kogito/addon/quarkus/common/reactive/messaging/MessageDecorator.java

File: runtime-services-modeling/deploying-services.adoc [hunk 6]
```diff
@@ -327,7 +327,7 @@ To use the DMN event-driven add-on, add the following code to the `pom.xml` file
 ////
 === {PRODUCT} explainability add-on
 
-The Explainability add-on in {PRODUCT} provides integration with Explainability and Trusty Services. You can also use https://github.com/kiegroup/kogito-examples/tree/stable/trusty-demonstration[`trusty-demonstration`] example application.
+The Explainability add-on in {PRODUCT} provides integration with Explainability and Trusty Services. You can also use https://github.com/apache/incubator-kie-kogito-examples/tree/stable/trusty-demonstration[`trusty-demonstration`] example application.
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:331]: https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/trusty-demonstration

File: runtime-services-modeling/deploying-services.adoc [hunk 7]
```diff
@@ -787,9 +787,9 @@ You can use the {PRODUCT} process event add-on to build a historical representat
...
 
-* https://github.com/kiegroup/kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/ProcessInstanceEventBody.java[ProcessInstanceEvent]: This event is published when a process instance is created, modified, or completed.
-// * https://github.com/kiegroup/kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/UserTaskInstanceEventBody.java[UserTaskInstanceEvent]: This event is published when a change occurs on a user task.
-* https://github.com/kiegroup/kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/VariableInstanceEventBody.java[VariableInstanceEvent]: This event is published when a variable is created, modified, or removed.
+* https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/ProcessInstanceEventBody.java[ProcessInstanceEvent]: This event is published when a process instance is created, modified, or completed.
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:790]: https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/ProcessInstanceEventMetadata.java

File: runtime-services-modeling/deploying-services.adoc [hunk 8]
```diff
@@ -787,9 +787,9 @@ You can use the {PRODUCT} process event add-on to build a historical representat
...
+// * https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/UserTaskInstanceEventBody.java[UserTaskInstanceEvent]: This event is published when a change occurs on a user task.
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:791]: https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/usertask/UserTaskInstanceEventMetadata.java

File: runtime-services-modeling/deploying-services.adoc [hunk 9]
```diff
@@ -787,9 +787,9 @@ You can use the {PRODUCT} process event add-on to build a historical representat
...
+* https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/VariableInstanceEventBody.java[VariableInstanceEvent]: This event is published when a variable is created, modified, or removed.
```
Comment (martinweiler) [inline runtime-services-modeling/deploying-services.adoc:792]: https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/api/kogito-events-api/src/main/java/org/kie/kogito/event/process/ProcessInstanceVariableEventBody.java

**Comment by martinweiler** [comment]
Quite a few broken links due to code refactoring. So while you are at it already for the repository change, I'd suggest to fix those as well @LightGuard

**Comment by LightGuard** [comment]
@martinweiler Please re-review.

**Review by martinweiler — APPROVED** [review]
LGTM @LightGuard

**Review by porcelli — CHANGES_REQUESTED** [review]
I think we should a downstream link to share the examples…

community is not using our product BOM, so I think we should publish under our monorepo the examples in the ibm org

**Comment by LightGuard** [comment]
Do we have such a link currently?

**Comment by porcelli** [comment]
> Do we have such a link currently?

Of course, not! :)

My comment implies that we need to get downstream examples available in the open :)

**Comment by LightGuard** [comment]
@porcelli feel free to create a ticket for that :)

**Comment by porcelli** [comment]
this is blocked by https://github.com/bamoe/private-bamoe-issues/issues/1591

**Review by veena-ux — COMMENTED** [review]
Reviewed and added few comments. In case of any queries, please reach out to me. Thanks.

File: runtime-services-modeling/deploying-decision-services.adoc [hunk 10]
```diff
@@ -360,11 +360,11 @@ To use the Jobs Management add-on, add the following code to the `pom.xml` file
 
 === {PRODUCT} Knative Eventing add-on
 
-The Knative Eventing add-on is used in the {PRODUCT} project for messaging. The https://github.com/kiegroup/kogito-runtimes/tree/main/addons/common/knative/eventing[{PRODUCT} Knative Eventing] add-on enables your project to connect to a https://knative.dev/docs/developer/eventing/sinks/[sink] or make it a https://github.com/knative-sandbox/eventing-kogito#kogito-source[KogitoSource].
+The Knative Eventing add-on is used in the {PRODUCT} project for messaging. The https://github.com/apache/incubator-kie-kogito-runtimes/tree/main/addons/common/knative/eventing[{PRODUCT} Knative Eventing] add-on enables your project to connect to a https://knative.dev/docs/developer/eventing/sinks/[sink] or make it a https://github.com/knative-sandbox/eventing-kogito#kogito-source[KogitoSource].
 
 The add-on processes the https://knative.dev/development/developer/eventing/sources/sinkbinding/[`K_SINK`] and https://knative.dev/development/developer/eventing/sources/sinkbinding/reference/#cloudevent-overrides[`K_CE_OVERRIDES`] environment variables injected by Knative Eventing controllers (`SinkBinding` or `KogitoSource`).
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply runtime-services-modeling/deploying-decision-services.adoc:365]: I don't believe adding the comma here is correct. There's a verb (processes) followed by two direct objects (K_SINK and K_CE_OVERRIDES) and the associated adjectives further describing the direct objects.

File: runtime-services-modeling/deploying-decision-services.adoc [hunk 11]
```diff
@@ -360,11 +360,11 @@ To use the Jobs Management add-on, add the following code to the `pom.xml` file
...
 
-The service requires the https://github.com/kiegroup/kogito-runtimes/tree/main/addons/common/messaging[{PRODUCT} Messaging] and https://quarkus.io/guides/reactive-messaging-http.html[Quarkus HTTP connector] libraries to wire the {PRODUCT} service with a given sink. The {PRODUCT} Messaging and Quarkus HTTP connector perform as dependencies for the Knative Eventing add-on.
+The service requires the https://github.com/apache/incubator-kie-kogito-runtimes/tree/main/addons/common/messaging[{PRODUCT} Messaging] and https://quarkus.io/guides/reactive-messaging-http.html[Quarkus HTTP connector] libraries to wire the {PRODUCT} service with a given sink. The {PRODUCT} Messaging and Quarkus HTTP connector perform as dependencies for the Knative Eventing add-on.
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply runtime-services-modeling/deploying-decision-services.adoc:367]: Yes, this is correct.

File: runtime-services-modeling/deploying-decision-services.adoc [hunk 12]
```diff
@@ -547,7 +547,7 @@ Any dependant add-on can implement the https://github.com/kiegroup/kogito-runtim
 . Enter the full name of your implementation class in the file.
 . Save the file.
 
-The `MessagePayloadDecoratorProvider` loads the file upon application startup and adds the file to the decoration chain. When {PRODUCT} calls the https://github.com/kiegroup/kogito-runtimes/blob/main/addons/common/messaging/common/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecoratorProvider.java[MessagePayloadDecoratorProvider#decorate], your implementation is part of the decoration algorithm.
+The `MessagePayloadDecoratorProvider` loads the file upon application startup and adds the file to the decoration chain. When {PRODUCT} calls the https://github.com/apache/incubator-kie-kogito-runtimes/blob/main/addons/common/messaging/src/main/java/org/kie/kogito/addon/cloudevents/message/MessagePayloadDecoratorProvider.java[MessagePayloadDecoratorProvider#decorate], your implementation is part of the decoration algorithm.
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply runtime-services-modeling/deploying-decision-services.adoc:550]: We could say "your implementation is added as part of the decoration algorithm," however, I do not believe that adds any further clarification for the target audience.

File: getting-started/orchestrating.adoc [hunk 13]
```diff
@@ -10,7 +10,7 @@ The following diagram illustrates how decisions are integrated with processes:
 .Embedded method to integrate a process with decisions in {PRODUCT}
 image::process-decisions-embedded-integration.png[Image of embedded method to integrate a process with decisions in {PRODUCT}]
 
-To view the complete project, you can refer to the https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
+To view the complete project, you can refer to the https://github.com/apache/incubator-kie-kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply getting-started/orchestrating.adoc:14]: Fixed

File: getting-started/orchestrating.adoc [hunk 14]
```diff
@@ -187,7 +187,7 @@ Then you can check the application Swagger UI at `http://localhost:8080/q/swagge
 
 For more {PRODUCT} examples that use decisions orchestrated by processes, see the following example applications in GitHub:
 
-* https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/process-business-rules-quarkus[`process-business-rules-quarkus`]: A quickstart project that shows the use of business rules and processes
-* https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/process-decisions-rest-quarkus[`process-decisions-rest-quarkus`]: An example project that shows the usage of decisions within processes. The focus here is to show how to integrate decisions in a remote way using REST APIs where they can be deployed decoupled from the process service.
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply getting-started/orchestrating.adoc:191]: Fixed

File: getting-started/orchestrating.adoc [hunk 15]
```diff
@@ -187,7 +187,7 @@ Then you can check the application Swagger UI at `http://localhost:8080/q/swagge
...
-* https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/process-decisions-rules-quarkus[`process-decisions-rules-quarkus`]: An example project that shows the usage of decisions within processes. The focus here is to show how to integrate decisions in an embedded way using Business Rule Task where they must be deployed together with the process.
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply getting-started/orchestrating.adoc:192]: Fixed

**Comment by LightGuard** [comment]
@veena-ux All the changes you requested have been addressed. If you are happy, please approve the changes here in GitHub so we can merge.

@porcelli You requested changes as well, until we have a downstream example repo, I am not able to incorporate your requests.

File: getting-started/orchestrating.adoc [hunk 16]
```diff
@@ -10,9 +10,17 @@ The following diagram illustrates how decisions are integrated with processes:
 .Embedded method to integrate a process with decisions in {PRODUCT}
 image::process-decisions-embedded-integration.png[Image of embedded method to integrate a process with decisions in {PRODUCT}]
 
+<<<<<<< HEAD
+To view the complete project, you can refer to the https://github.com/apache/incubator-kie-kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
+||||||| parent of ed92dc1 (Addressing @veena-ux comments in orchestrating.adoc)
 To view the complete project, you can refer to the https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
+=======
+To view the complete project, se the https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
+>>>>>>> ed92dc1 (Addressing @veena-ux comments in orchestrating.adoc)
```
Comment (jomarko) [inline getting-started/orchestrating.adoc:19]: I think here we miss resolving merge conflict.

**Review by jomarko — COMMENTED** [review]
Just one comment about unresolved merge conflict.

**Comment by LightGuard** [comment]
@jomarko fixed

**Review by baldimir — APPROVED** [review]

File: installation/canvas.adoc [hunk 17]
```diff
@@ -139,12 +139,14 @@ By default, {CANVAS} is configured to fetch this image from https://quay.io/repo
 
 [IMPORTANT]
 ====
-The pre-configured image is NOT part of BAMOE, and comes from the open-source KIE community. People installing {CANVAS} must change `KIE_SANDBOX_DMN_DEV_DEPLOYMENT_BASE_IMAGE_URL` to make sure the used image meets their requirements. For a reference on how to build your own image that is compatible with {CANVAS} Dev deployments, please go to the KIE community's source code for `dmn-dev-deployment-base-image` at https://github.com/kiegroup/kie-tools/tree/{KIE_TOOLS_COMMUNITY_VERSION}/packages/dmn-dev-deployment-base-image
+The pre-configured image is NOT part of {PRODUCT_SHORT}, and resides in the open-source Apache KIE community.
+If you are  installing {CANVAS}, you must change `KIE_SANDBOX_DMN_DEV_DEPLOYMENT_BASE_IMAGE_URL` to make sure the used image meets your requirements.
+For a reference on how to build your own image that is compatible with {CANVAS} development deployments, please go to the Apache KIE community's source code for `dmn-dev-deployment-base-image` at https://github.com/kiegroup/kie-tools/tree/{KIE_TOOLS_COMMUNITY_VERSION}/packages/dmn-dev-deployment-base-image.
```
Comment (jomarko) [inline installation/canvas.adoc:144]: Shouldn't we use `apache` also here? I mean both links below works while the later one seems to be newer. 
- https://github.com/kiegroup/kie-tools/tree/0.31.0/packages/dmn-dev-deployment-base-image
- https://github.com/apache/incubator-kie-tools/tree/0.31.0/packages/dmn-dev-deployment-base-image

**Review by jomarko — COMMENTED** [review]
One observation about `kiegroup` vs `apache`

File: installation/canvas.adoc (hunk 17 above)
Comment (LightGuard) [reply installation/canvas.adoc:144]: Yes, we should, good catch.

Comment (LightGuard) [reply installation/canvas.adoc]: Fixed

File: getting-started/orchestrating.adoc [hunk 18]
```diff
@@ -10,7 +10,7 @@ The following diagram illustrates how decisions are integrated with processes:
 .Embedded method to integrate a process with decisions in {PRODUCT}
 image::process-decisions-embedded-integration.png[Image of embedded method to integrate a process with decisions in {PRODUCT}]
 
-To view the complete project, you can refer to the https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
+To view the complete project, you can refer to the https://github.com/apache/incubator-kie-kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.
 
 The example of a traffic violations project describes how to use decisions within the processes and how to integrate decisions in an embedded way using the business rule tasks, which must be deployed with a process in the same application.
```
[the remark this answers is not in the document]
Comment (veena-ux) [reply getting-started/orchestrating.adoc]: > The example of a traffic violations project describes how to use decisions within the processes and integrate decisions in an embedded form using the business rule tasks that must be deployed with a process in the same application.
> 
> Comment: The sentence is too long. You may think of breaking it into two sentences at least.

Much better.
--------------
In the previous sentence No.13, you may replace refer with see
like...
To view the complete project, see https://github.com/apache/incubator-kie-kogito-examples/tree/main/kogito-quarkus-examples/process-decisions-quarkus[`process-decisions-quarkus`] example application for Quarkus.

File: getting-started/orchestrating.adoc [hunk 19]
```diff
@@ -187,7 +189,7 @@ Then you can check the application Swagger UI at `http://localhost:8080/q/swagge
...
+* https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/process-business-rules-quarkus[`process-business-rules-quarkus`]: An example project that shows the use of business rules and processes
+* https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/process-decisions-rest-quarkus[`process-decisions-rest-quarkus`]: An example project that shows the usage of decisions within processes. The project focuses on how to integrate decisions in a remote way using REST APIs where they can be deployed and decoupled from the process service.
+* https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/process-decisions-rules-quarkus[`process-decisions-rules-quarkus`]: An example project that shows the usage of decisions within processes. The project focuses on integrating decisions in an embedded way using Business Rule Tasks where they must be deployed together with the process.
+* https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/process-error-handling[`process-error-handling`]: An usage scenario of the Error Handling Strategy
```
Comment (kbowers-ibm) [inline getting-started/orchestrating.adoc:195]: ...A usage scenario...

**Review by kbowers-ibm — APPROVED** [review]
Looks like this is ready to be merged, thanks for the PR!

**Comment by LightGuard** [comment]
Thanks everyone for the help!

**Pull request merged**